data-prepper-plugins/otel-metrics-raw-processor/src/main/java/com/amazon/dataprepper/plugins/processor/otelmetrics/model/RawHistogram.java

```diff
@@ -107,27 +107,24 @@ public String toJson() throws JsonProcessingException {
 
         private final double lowerBound;
         private final double upperBound;
-        private final long numberOfObservations;
+        private final long count;
 
-        public Bucket(double lowerBound, double upperBound, long numberOfObservations) {
+        public Bucket(double lowerBound, double upperBound, long count) {
             this.lowerBound = lowerBound;
             this.upperBound = upperBound;
-            this.numberOfObservations = numberOfObservations;
+            this.count = count;
         }
 
-        @JsonProperty("lo")
         public double getLowerBound() {
```

Comment:
Thanks for renaming `count`. Why did you rename `lo` and `hi` to `lowerBound` and `upperBound`? I believe OTel uses `min` and `max` and these are both smaller for the OpenSearch documents.